src/components/Editor/TextEditor.tsx

```diff
@@ -60,28 +60,26 @@ const TextEditor = (): JSX.Element => {
           <SaveAltIcon sx={{ fontSize: 30 }} />
         </IconButton>
       </Header>
-
       <Box
         sx={{
           m: "1em",
           mt: "0.5em",
           mb: "1.2em",
           mr: "1.3em",
           alignItems: "center",
-          height:"100%",
+          height: "100%",
         }}
       >
-        <textarea 
+        <textarea
           style={{
             background: "white",
             borderRadius: "5px",
             width: "100%",
-            height:"100%",
+            height: "100%",
```

Comment:
when i was working on it, i don't really understand the meaning of 100%. is there any way to use inherit in this type of styling?

Reply:
the problem with inherit is that it gets the width of the parent so the child component will go out of parent. 100% tells the component to take the available spacing but be contained in parent